Sources/eppo/ConfigurationRequester.swift

```diff
@@ -14,3 +18,15 @@ class ConfigurationRequester {
         return try UniversalFlagConfig.decodeFromJSON(from: String(data: urlData, encoding: .utf8)!);
     }
 }
+
+class JsonConfigurationRequester: ConfigurationRequesterProtocol {
+    private let configurationJson: String;
+
+    public init(configurationJson: String) {
+        self.configurationJson = configurationJson
+    }
+
+    public func fetchConfigurations() throws -> UniversalFlagConfig {
+        return try UniversalFlagConfig.decodeFromJSON(from: configurationJson);
+    }
```

Comment:
minor: I was looking to see if this would get triggered every 30 seconds by configuration poller and it looks like iOS SDK doesn't have the poller at all. Is this intentional? If so, we should update the README as it says the SDK does poll the configuration every 30s

Reply:
That's correct there is no included poller. Ok good idea I'll make a note.